src/main/java/edu/ucsb/cs156/example/entities/UCSBOrganization.java

```diff
@@ -12,7 +12,7 @@
 @AllArgsConstructor
 @NoArgsConstructor
 @Builder
-@Entity(name = "ucsborganization")
+@Entity(name = "UCSBOrganization")
```

Comment:
This could mess up the prod database because it uses this for the name of the table. I'm not sure if it's case sensitive but I don't want to find out.